Stripe/PublicHeaders/STPPaymentCardTextField.h

```diff
@@ -36,7 +36,7 @@ IB_DESIGNABLE
 @property (nonatomic, copy, null_resettable) UIFont *font UI_APPEARANCE_SELECTOR;
 
 /**
- The text color to be used when entering valid text. Default is [UIColor blackColor]. 
+ The text color to be used when entering valid text. Default is [UIColor labelColor].
```

Comment:
possibly worth documenting both iOS 13+ and pre-iOS 13 defaults in here?

Reply:
I did in my first draft, but I took it out for simplicity. I can't think of a situation where a developer would be confused by this, but I can put it back in if you think it would help.

Reply:
👍 eh I think it's fine to ship as-is